tests/lax_test.py

```diff
@@ -1651,28 +1652,38 @@ def testConvWithGeneralPaddingGrad(self, lhs_shape, rhs_shape, dtype, strides,
            [(1, 1), (2, 1)],  # lhs_dils
            [(1, 1), (2, 2)])  # rhs_dils
           for b, i, j in itertools.product([1, 2], repeat=3)]
+      for feature_group_count in [1]  # TODO(mattjj): feature_group_count=2
```

Comment:
Could you change this line to `for feature_group_count in [1, 2]` and check if those tests pass? Since `vmap(conv)` can generate calls with `feature_group_count > 1` now, we need to make sure we can differentiate with `feature_group_count > 1` to support `grad(vmap(conv))`.